frontend/src/components/core/EditableCode.tsx

```diff
@@ -36,9 +36,9 @@ const EditableCode = React.memo(({
                         }} language={langage} />
                         : <ReactMarkdown
                             components={{
-                                h1: ({ ...props }) => (<p className="text-xl underline" {...props} />),
-                                h2: ({ ...props }) => (<p className="text-md underline" {...props} />),
-                                p: ({ ...props }) => (<p className="text-sm" {...props} />),
+                                h1: ({ node, ...props }) => (<p className="text-xl underline" {...props} />),
+                                h2: ({ node, ...props }) => (<p className="text-md underline" {...props} />),
+                                p: ({ node, ...props }) => (<p className="text-sm" {...props} />),
                             }}
                             children={code || "Nothing"}
                             disallowedElements={["img", "script"]}
```

Comment:
should we disallow any element we do not style?

Reply:
I was more focused on disallowing elements that can cause XSS execution